shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapterTest.java

```diff
@@ -125,7 +130,8 @@ public void init() {
 
   @Test
   public void createTableEvent() {
-    when(messageReader.read()).thenReturn(Optional.of(apiaryCreateTableEvent));
+    when(messageReader.read()).thenReturn(Optional.of(new MessageEvent(apiaryCreateTableEvent,
```

Comment:
this is getting a bit hard to read maybe:
```
when(messageReader.read()).thenReturn(newMessageEvent(apiaryCreateTableEvent);
```
where:
```
private Optional<MessageEvent> newMessageEvent(ListenerEvent event) {
    return Optional.of(new MessageEvent(apiaryCreateTableEvent,Collections.singletonMap(SqsMessageProperty.SQS_MESSAGE_RECEIPT_HANDLE, RECEIPT_HANDLE)));
}
```

Reply:
Yep much cleaner, thanks!